triage/2020-08-31.md

```diff
@@ -0,0 +1,36 @@
+# 2020-08-31 Triage Log
+
+See the [README](README.md) for details on how this log was gathered.
+
+A quiet week for the most part with the exception of two PRs. The first,
+[#76027](https://github.com/rust-lang/rust/pull/76027), was a rather large
+regression that was soon fixed by
+[#76030](https://github.com/rust-lang/rust/pull/76030). The net result seems to
+be a small improvement overall on optimized and debug builds.
+
+Triage done by @ecstaticmorse.
+Revision range: [03017003c77d782cf7ed841a05d7c628a9b93f25..d927e5a655809b6b20501889e093c085d6ffe6f7][range]
+
+[range]: https://perf.rust-lang.org/?start=03017003c77d782cf7ed841a05d7c628a9b93f25&end=d927e5a655809b6b20501889e093c085d6ffe6f7&absolute=false&stat=instructions%3Au
+
+1 Regression, 2 Improvements, 0 of them in rollups.
+
+#### Regressions
+
+[#76027](https://github.com/rust-lang/rust/pull/76027) ty: remove obsolete pretty printer
+- [A major regression on some `incr-patched` benchmarks](https://perf.rust-lang.org/compare.html?start=e98f0632bbec24b196dbd6fc820537f6f3724807&end=8ed5cb56b5e5cc216eb6820a44dd4f7ef65107b0&stat=instructions:u)
+- Very small regressions on full builds of other benchmarks along with a major one on `await-call-tree`
+- Seems to have been solved by #76030 (see below).
+
+#### Improvements
+
+[#75754](https://github.com/rust-lang/rust/pull/75754) Switch to Snappy compression for metadata
+- [A small improvement across the board](https://perf.rust-lang.org/compare.html?start=fe8ab8a530fc2369e2748421a319444383363340&end=7fc048f0712ba515ca11fac204921b9ad8a0c5a3&stat=instructions:u)
+
+[#76030](https://github.com/rust-lang/rust/pull/76030) cg\_llvm: `fewer_names` in `uncached_llvm_type`
+- [A major improvement on some `incr-patched` benchmarks](https://perf.rust-lang.org/compare.html?start=897ef3a0ec001b89fffe7125c20d6a6bb12fee6c&end=1d22f75c9f75cad2e408a145861904898ac982dd&stat=instructions:u)
+- Small improvements on full builds of other benchmarks along with a major one on `await-call-tree`
+
+#### Nags requiring follow up
+
+None
```

Comment:
I think it would be good to follow-up on the regression to make sure we're not missing a bunch of wine (i.e. if it's coincidental that the two PRs have similar effects).

Reply:
I already left a comment on the PR, but given [this comment](https://github.com/rust-lang/rust/pull/76030#issuecomment-683245601) I'm pretty sure we won't need to nag anybody.

Reply:
Ah, great, ok.